docs/api/benchmarks.rst

```diff
@@ -19,6 +19,9 @@ Quality
 
    Qualitative evaluation of word tokenization.
 
+   **Disclaimer**: our current methodology is based on a unpublished work; hence,
```

Comment:
Can you add to `2.1 branch`?

Reply:
sry, what do you mean?

Reply:
> 
> 
> sry, what do you mean?

Create a pull request to add a disclaimer to the documents of 2.1.